html-style-guide.md

```diff
@@ -41,6 +41,10 @@ Aligned:
 </a>
 ```
 
+# Identification
+
+- Use `class` for identifying elements. Don't use `id`.
```

Comment:
How about replacing this line with "Prefer `class` for identifying elements that need to be selected for styling or queried via JavaScript."